support-frontend/assets/helpers/globals.js

```diff
@@ -25,29 +25,37 @@ const emptyAmountsSettings = {
   ANNUAL: [],
 };
 
-const getSettings = (): Settings => getGlobal('settings') || {
-  switches: {
-    experiments: {},
-  },
-  amounts: {
-    GBPCountries: emptyAmountsSettings,
-    UnitedStates: emptyAmountsSettings,
-    EURCountries: emptyAmountsSettings,
-    AUDCountries: emptyAmountsSettings,
-    International: emptyAmountsSettings,
-    NZDCountries: emptyAmountsSettings,
-    Canada: emptyAmountsSettings,
-  },
-  contributionTypes: {
-    GBPCountries: [],
-    UnitedStates: [],
-    EURCountries: [],
-    AUDCountries: [],
-    International: [],
-    NZDCountries: [],
-    Canada: [],
-  },
-  metricUrl: '',
+const getSettings = (): Settings => {
+  const globalSettings = getGlobal('settings');
+  if (globalSettings) {
+    const useDigitalVoucher = getGlobal('useDigitalVoucher');
+    return { ...globalSettings, useDigitalVoucher };
```

Comment:
why not just put `useDigitalVoucher` in `window.guardian.settings.useDigitalVoucher` rather than separately then having the logic to copy it over.
I'm not clear of the purpose of the default/null values below either - may be just a client side thing I don't understand though!

Reply:
@rupertbates - are you able to explain this process? I don't feel confident that I would explain it correctly!

Reply:
if there's a good reason then don't worry, I just wanted to flag it as looking a bit at odds with the rest of the code